Week2/6.js

```diff
@@ -0,0 +1,12 @@
+const myArray = [];
+
+console.log('The value of myArray is undefined');
```

Comment:
I think the value is not `undefined`, but an `empty array`. What do you think :)?

Reply:
Right! These code lines belong to where I was just diving into Pandora's box :)